util/concurrent.go

```diff
@@ -0,0 +1,68 @@
+package util
```

Comment:
I feel a bit uncomfortable introducing this package just for this purpose since people may start putting everything here going forward. Given that we use this only for database connections at the moment, how about moving this to `sqldef/sqldef/database` package (e.g. `database/concurrent.go`) instead?

Reply:
I moved it to `database/concurrent.go` 👍